entries/docs/SWC-136.md

```diff
@@ -60,6 +72,18 @@ contract OddEven {
 }
 ```
 
+#### Comments
+
+The vulnerable version above requires the players to send the number they are using
+as part of the transaction.
+This means the first player's number will be visible, allowing the second player to select a number
+that they know will make them a winner.
+(This assumption is simplistic to illustrate - there are also possibilities to front-run players,
+among other potential issues).
+
+In the fixed version below, the players instead submit a commitment that encrypts their number,
```

Comment:
```suggestion
In the fixed version below, the players instead submit a commitment that obfuscates their number,
```

Better: Obfuscate since hashing data with a blinding factor to prevent someone else finding the pre-image isn't really encryption. Thought about using "encode" but that doesn't reflect the fact that the data can't be decoded by anyone until the reveal.